This does not compile:
```
#23 618.6 In file included from /opt/software/linux-x86_64_v2/glm-1.0.1-hnfgp474nuaxdv7qw6frort4qs43zret/include/glm/gtx/string_cast.hpp:21,
#23 618.6                  from /tmp/root/spack-stage/spack-stage-eic-opticks-0.1.0-vpxzp2ibrbzri4w4kygkswwctucjcy2j/spack-src/sysrap/s_tv.h:16,
#23 618.6                  from /tmp/root/spack-stage/spack-stage-eic-opticks-0.1.0-vpxzp2ibrbzri4w4kygkswwctucjcy2j/spack-src/sysrap/s_tv.cc:1:
#23 618.6 /opt/software/linux-x86_64_v2/glm-1.0.1-hnfgp474nuaxdv7qw6frort4qs43zret/include/glm/gtx/dual_quaternion.hpp:24:9: error: #error "GLM: GLM_GTX_dual_quaternion is an experimental extension and may change in the future. Use #define GLM_ENABLE_EXPERIMENTAL before including it, if you really want to use it."
#23 618.6    24 | #       error "GLM: GLM_GTX_dual_quaternion is an experimental extension and may change in the future. Use #define GLM_ENABLE_EXPERIMENTAL before including it, if you really want to use it."
#23 618.6       |         ^~~~~
```

--- a/spack_repo/eic/packages/eic_opticks/package.py
+++ b/spack_repo/eic/packages/eic_opticks/package.py
@@ -36,3 +36,9 @@ class EicOpticks(CMakePackage, CudaPackage):
     depends_on("openssl")
     depends_on("plog")
     depends_on("python")
+
+    def setup_build_environment(self, env):
+        # GLM 0.9.9+ requires this for experimental GTX headers such as
+        # dual_quaternion, which are reached via string_cast in this codebase.
+        if self.spec.satisfies("^glm@0.9.9:"):
+            env.append_flags("CPPFLAGS", "-DGLM_ENABLE_EXPERIMENTAL")

